tmk_core/common/keyboard.c

```diff
@@ -62,19 +62,36 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #endif
 
 
-
 #ifdef MATRIX_HAS_GHOST
+extern const uint16_t keymaps[][MATRIX_ROWS][MATRIX_COLS];
+// bit map of true keys and empty spots in matrix, each row is reversed
+static uint16_t get_row_ghost_check(uint16_t row){
+    for (int col = 0; col < MATRIX_COLS; col++) {
+        if (keymaps[0][row][col])
```

Comment:
I updated it in a rush, all better now. A bit smaller as well. Previous: 20662, after fix: 20858

I'd like to add, this is an important feature and it takes up zero space unless you need it. If you do need it, you likely are using an old keyboard that doesn't need many advanced byte hogging features.